### PR TITLE
Add botpress configs

### PR DESCRIPTION
**Pull request by amiraliemami**
Reviewer: @suzinyou 

Estimate: 30min

----

# Add botpress v12 config and instructions

## Ticket

Implements: [171](https://idinsight.atlassian.net/browse/AAQ-171?atlOrigin=eyJpIjoiNWNlZDdmNmI3NDE2NGM4NGEwNTVlODFhNzIzZGY1ZTciLCJwIjoiaiJ9).
See demo video [here](https://idinsight.slack.com/archives/C05RSDLN1GA/p1700231501635329).

## Description, Motivation and Context

Added local installation and running instructions for Botpress v12 (OSS) chat manager and a zipped chatbot flow that could be loaded for a demo chatbot that uses AAQ for FAQ fetching.

## How Has This Been Tested?

Follow `chat_managers/botpress_v12/README.md`.

## To do

- Add instruction and configs for connecting botpress to whatsapp through twilio
- Add deployment design/instructions for botpress
- Write our own Twilio-less WhatsApp connector if doable

## Checklist:

This checklist is a useful reminder of small things that can easily be forgotten.
Put an `x` in all the items that apply and remove any items that are not relevant to this PR.

- [-] My code follows the style guidelines of this project
- [-] I have reviewed my own code to ensure good quality
- [-] I have tested the functionality of my code to ensure it works as intended
- [x] I have resolved merge conflicts
- [-] I have updated the automated tests (if applicable)
- [x] I have written [good commit messages][1]
- [x] I have updated the README file (if applicable)
- [-] I have updated affected documentation (if applicable)

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Review by suzinyou — CHANGES_REQUESTED** [review]
@amiraliemami  Thank you for this ! Was able to use the emulator. I tried to test with dev server endpoints but didn't work because of the SSL certificate issue.

I've made a few small comments. I think you may have deleted some files by mistake?

File: docs/chat_managers/botpress_v12.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+        docker pull botpress/server
+
+2. Run the image
+
+        docker run -d --name=botpress -p 3000:3000 botpress/server
+
+    ??? note "Issues running the Docker image on Macs with Apple Silicon?"
+
+        If you're having issues with image type on MacOS, you can also try running the image through the Docker UI instead of commandline. This fixed the issue for me.
```
Comment (suzinyou) [inline docs/chat_managers/botpress_v12.md:23]: How about we remove this? It is not actually an issue just a warning!

```suggestion
```

**Review by suzinyou — COMMENTED** [review]
Thank you!! Let one suggestion. I also got it to work with our dev server :) Very neat.

Future steps: to deploy this, we should figure out how 1. include botpress configs in version control and 2. to load the chat flow configs in the volume or in an image of botpress

File: docs/chat_managers/botpress_v12.md (hunk 1 above)
Comment (amiraliemami) [reply docs/chat_managers/botpress_v12.md:23]: Oh yeah you're right - it does run for me too but with a warning

**Pull request merged**